storage/backend/ceph/internals.go

```diff
@@ -1,26 +1,69 @@
 package ceph
 
 import (
+	"encoding/json"
 	"fmt"
 	"os/exec"
 	"regexp"
 	"strings"
 
-	"encoding/json"
-
 	"github.com/contiv/volplugin/storage"
 
 	log "github.com/Sirupsen/logrus"
 )
 
+func (c *Driver) lockImage(do storage.DriverOptions) error {
+	poolName := do.Volume.Params["pool"]
+
+	if err := exec.Command("rbd", "lock", "add", do.Volume.Name, do.Volume.Name, "--pool", poolName).Run(); err != nil {
```

Comment:
should we use `Output()` or `CombinedOutput()` here instead of `Run()`? It might be useful to log command output in case of failures.